raft/log.go

```diff
@@ -1045,7 +1054,7 @@ func (l *Log) candidateLoop(closing <-chan struct{}) State {
 			l.leaderID = l.id
 			l.unlock()
 			return Leader
-		case ch := <-l.Clock.AfterElectionTimeout():
+		case ch := <-electionTimer.C():
 			close(ch)
```

Comment:
The `ch` is returned from `C() chan chan struct{}`. It's used to notify the sender that the operation is complete. It's used for testing.